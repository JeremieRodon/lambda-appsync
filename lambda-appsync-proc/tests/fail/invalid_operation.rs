use lambda_appsync::{appsync_lambda_main, appsync_operation, AppsyncError};

appsync_lambda_main!("../../../../schema.graphql");

#[appsync_operation(query(nonexistent))]
async fn invalid_operation() -> Result<String, AppsyncError> {
    Ok("test".to_string())
}
