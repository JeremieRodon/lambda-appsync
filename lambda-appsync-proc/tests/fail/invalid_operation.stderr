error[E0576]: cannot find method or associated constant `query_nonexistent` in trait `crate::DefautOperations`
 --> tests/fail/invalid_operation.rs:5:27
  |
5 | #[appsync_operation(query(nonexistent))]
  |                           ^^^^^^^^^^^ not found in `crate::DefautOperations`
